udata_front/settings.py

```diff
@@ -33,3 +33,11 @@
 GUIDES_ORGANIZATION_URL = 'https://guides.etalab.gouv.fr/data.gouv.fr/creer-rejoindre-organisation/'
 GUIDES_DATASET_URL = 'https://guides.etalab.gouv.fr/data.gouv.fr/publier-jeu-de-donnees/'
 GUIDES_REUSE_URL = 'https://guides.etalab.gouv.fr/reutilisation/publier-reutilisation/'
+
+# CAPTCHEtat
+CAPTCHETAT_BASE_URL = 'https://sandbox-api.piste.gouv.fr/piste/captcha'
```

Comment:
I think we should document the behaviour of `CAPTCHETAT_BASE_URL` to activate or deactivate captchetat.
We should also add a comment on additional `CAPTCHETAT_CLIENT_ID` and `CAPTCHETAT_CLIENT_SECRET`.

Maybe adding it in settings.py + in changelog would do :)

Reply:
Done !